lib/MetaCPAN/Web/Controller/Release.pm

```diff
@@ -158,6 +165,56 @@ sub view : Private {
     );
 }
 
+sub _files_to_categories {
```

Comment:
For the sake of consistency, it would be nice not to mix functions and methods in the same class.  So, we could either call this via `$self->` or we could move it into some kind of a helper class that exports this function.